client/src/dialogs/schema-editor-dialog.js

```diff
@@ -0,0 +1,20 @@
+import { inject } from "aurelia-framework";
+import { DialogController } from "aurelia-dialog";
+
+@inject(DialogController)
+export class SchemaEditorDialogImplementation {
+  constructor(dialogController) {
+    this.dialogController = dialogController;
+  }
+
+  activate(config) {
+    this.config = config;
+    delete this.config.schema["Q:options"].openInDialog;
```

Comment:
This needs to be deleted to not end up in an endless loop right? Can you add a comment about this? I only got it on the 2nd thought.